### DISCUSSION TITLE
【测试】大图片

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1353] [其他]
26MB图片测试
<a href="https://i.ibb.co/1X8J41V/torino.png">
<img src="https://i.ibb.co/1X8J41V/torino.png" alt="torino" border="0" />
</a>

**Comment by lllliufeng** [comment]
图片挂了？

**Comment by capacitor1** [comment]
好像还是无法直接放图片。。

只能引用到外部

**Comment by capacitor1** [comment]
https://i.ibb.co/1X8J41V/torino.png

这是我使用的图片链接，GitHub好像遇到引用外部链接的图片就会自动下载到自己服务器上
